examples/dedx_get_simple_stp_with_program.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+#include <dedx.h>
+
+int main(int argc, char *argv[]) {
+    int err;
+    float energy = 100.0;
+    int program = DEDX_BETHE_EXT00;
+    int material = DEDX_HYDROGEN;
+    int target = DEDX_WATER_LIQUID;
+
+    float result = dedx_get_simple_stp_for_program(program, material, target, energy , &err);
```

Comment:
This function is defined as:
```
dedx_get_simple_stp_for_program(const int program, const int ion, const int target, float energy, int *err)
```

why do you use here `material` instead of `ion` as second argument ?